pkg/operator/ceph/csi/template/cephfs/csi-cephfsplugin-provisioner-dep.yaml

```diff
@@ -212,6 +226,10 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /csi
+            {{ if .CSILogRotation }}
+            - mountPath: {{ .PrefixCsiLogDirPath }}/log/{{ .CSILogFolder }}
```

Comment:
This is confusing to have a "PrefixCsiLogDirPath", and a "CSILogFolder". Is the latter more of a component name? What about calling it `.CSILogComponent`? And maybe call the other one `.CsiLogParentDir`? The "log" subdir is appended, so it seems more like the parent directory of the logs.

Reply:
Updated, 
Path: `/var/lib/rook/rook-ceph.rbd.csi.ceph.com/log/controller-plugin`

`{{ .CSILogFolder }}` is the deployment or daemon set name, do we also want to change it to `CSILogComponent`??